ethcore/evm/src/interpreter/mod.rs

```diff
@@ -342,6 +346,11 @@ impl<Cost: CostType> Interpreter<Cost> {
 
 				// Calculate gas cost
 				let requirements = self.gasometer.as_mut().expect(GASOMETER_PROOF).requirements(ext, instruction, info, &self.stack, self.mem.size())?;
+				let total_op_cost = gas_profile.get(&info.name.to_string()).cloned();
```

Comment:
I don't think that you really need to `cloned` here since nothing takes ownership over the interior of `total_op_cost`. Try `as_ref` instead. Also I think that `info.name` (no to_string) will also work since `HashMap.get` doesn't take ownership over the argument